README.md

```diff
@@ -20,6 +20,25 @@ serve SSH and HTTPS on the same port.
 Compile and install
 ===================
 
+Docker container
+----------------
+
+For easy installation and usage you can use unofficial docker container.
+Container built with `libconfig`, `libwrap` and `libcap` support. 
+It can be found on [Docker Hub](https://hub.docker.com/r/riftbit/sslh/)
+
+**Example with --net host**
```

Comment:
Can you add a description of what "--net host" means? As a non-docker user, I have no idea what the difference is between this setting and the next "exposed ports" setting (I guess the latter means it opens ports on the outside).

Reply:
It have difference in network type. if set `--net host` - container will work with host network interface directly, like you run application in host system, and all ports exposed. If this option not setted container will work with bridged network interface, and you will need to expose all ports and make additional configurations to allow container work with other addresses.